frontend/src/scenes/surveys/SurveyView.tsx

```diff
@@ -152,15 +152,27 @@ export function SurveyView({ id }: { id: string }): JSX.Element {
                                               {(survey.questions[0].type === SurveyQuestionType.SingleChoice ||
                                                   survey.questions[0].type === SurveyQuestionType.MultipleChoice) && (
                                                   <div className="mb-4">
-                                                      <Query
-                                                          query={{
-                                                              kind: NodeKind.DataTableNode,
-                                                              source: {
-                                                                  kind: NodeKind.HogQLQuery,
-                                                                  query: `select count(), arrayJoin(JSONExtractArrayRaw(properties, '$survey_response')) as choice from events where event == 'survey sent' and properties.$survey_id == '${survey.id}' group by choice order by count() desc`,
-                                                              },
-                                                          }}
-                                                      />
+                                                      {survey.questions[0].type === SurveyQuestionType.SingleChoice ? (
+                                                          <Query
+                                                              query={{
+                                                                  kind: NodeKind.DataTableNode,
+                                                                  source: {
+                                                                      kind: NodeKind.HogQLQuery,
+                                                                      query: `select count(), properties.$survey_response as choice from events where event == 'survey sent' and properties.$survey_id == '${survey.id}' group by choice order by count() desc`,
```

Comment:
suggestion: extract query out to the logic that generates the right one and then populate here in one Query component

Reply:
yeaaaaa that's what I've been doing, got lazy with this one, will do when I refactor <Group too lol

Reply:
(of course this is also affecting users so I would pref to get this in asap first and then make improvements, not just being lazy here)